content/en/docs/setup/_index.md

```diff
@@ -12,84 +12,101 @@ content_template: templates/concept
-manage and operate your clusters so you don’t have to.  
-
-You should pick a hosted solution if you:
-
-* Want a fully-managed solution
-* Want to focus on developing your apps or services  
-* Don’t have dedicated site reliability engineering (SRE) team but want high availability
-* Don't have resources to host and monitor your clusters 
-
-Pick a [hosted solution](/docs/setup/pick-right-solution/#hosted-solutions).
-
-## Turnkey – Cloud Solutions
-
-
-These solutions allow you to create Kubernetes clusters with only a few commands and 
-are actively developed and have active community support. They can also be hosted on 
-a range of Cloud IaaS providers, but they offer more freedom and flexibility in 
-exchange for effort. 
-
-You should pick a turnkey cloud solution if you:
-
-* Want more control over your clusters than the hosted solutions allow
-* Want to take on more operations ownership 
-
-Pick a [turnkey cloud solution](/docs/setup/pick-right-solution/#turnkey-cloud-solutions)
-
-## Turnkey – On-Premises Solutions
-
-These solutions allow you to create Kubernetes clusters on your internal, secure,
-cloud network with only a few commands.
-
-You should pick a on-prem turnkey cloud solution if you:
-
-* Want to deploy clusters on your private cloud network
-* Have a dedicated SRE team
-* Have the resources to host and monitor your clusters
-
-Pick an [on-prem turnkey cloud solution](/docs/setup/pick-right-solution/#on-premises-turnkey-cloud-solutions).
-
-## Custom Solutions
-
-Custom solutions give you the most freedom over your clusters but require the 
-most expertise. These solutions range from bare-metal to cloud providers on 
-different operating systems.
-
-Pick a [custom solution](/docs/setup/pick-right-solution/#custom-solutions).
+## Learning environment
+
+If you want to get familiar with Kubernetes, you can use the Docker-based solutions. You can use tools that are supported by the Kubernetes community or the tools in the ecosystem to setup a Kubernetes cluster on a local-machine. 
+
+{{< table caption="Local-machine solutions table that lists the tools supported by the community and the ecosystem to deploy Kubernetes." >}}
+
+|Community           |Ecosystem     |
+| ------------       | --------     |
+| [Minikube](/docs/setup/minikube/) | [CDK on LXD](https://www.ubuntu.com/kubernetes/docs/install-local) |
+| [Kubeadm-dind](https://github.com/kubernetes-sigs/kubeadm-dind-cluster) | [Docker Desktop](https://www.docker.com/products/docker-desktop)|
+| [Kubernetes IN Docker](https://github.com/kubernetes-sigs/kind) | [Minishift](https://docs.okd.io/latest/minishift/)|
+|                     | [MicroK8s](https://microk8s.io/)|
+|                     | [IBM Cloud Private-CE (Community Edition)](https://github.com/IBM/deploy-ibm-cloud-private) |
+|                     | [IBM Cloud Private-CE (Community Edition) on Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers)|
```

Comment:
nit: I realize the second `IBM Cloud Private-CE` entry is on the 1.14 version, but is this a link to a private repo? Will the content be updated?

Reply:
Thanks @kbhawkey !!!! This was something that I was contemplating before adding. I was not sure if the ecosystem still supports it.